src/start.sh

```diff
@@ -126,7 +126,11 @@ start() {
   sh /opt/pihole/pihole-FTL-prestart.sh
   capsh --user=$DNSMASQ_USER --keep=1 -- -c "/usr/bin/pihole-FTL $FTL_CMD >/dev/null" &
 
-  tail -f /var/log/pihole-FTL.log &
+  if [ "${TAIL_FTL_LOG:-1}" -eq 1 ]; then
+    tail -f /var/log/pihole-FTL.log &
```

Comment:
You might want to change to

```suggestion
    tail -f /var/log/pihole/FTL.log &
```

As the other is only a simlink we wanted to remove with v6. See

https://github.com/pi-hole/pi-hole/blob/fef2861eae23442ad823d51c9b4bfdd5b38ab200/advanced/Templates/pihole-FTL-prestart.sh#L29-L34

Reply:
Good point well made